app/helpers/utils.py

```diff
@@ -36,3 +44,47 @@ def get_logging_cfg():
 def init_logging():
     config = get_logging_cfg()
     logging.config.dictConfig(config)
+
+
+def prevent_erroneous_kml(kml_string):
+    kml_string = re.sub(r'on\w*=(".+?"|\'.+?\')', '', kml_string, flags=re.IGNORECASE)
+    kml_string = re.sub(
+        r'(<|&lt;)\s*\bscript\b.*?(>|&gt;).*?(<|&lt;)/\s*\bscript\b\s*(>|&gt;)',
+        ' ',
+        kml_string,
+        flags=re.IGNORECASE | re.DOTALL
+    )
+    return kml_string
+
+
+def validate_content(content):
+
+    def inner_decorator(func):
+
+        @wraps(func)
+        def wrapped(*args, **kwargs):
+            if request.content_type != content:
+                abort(make_error_msg(415, 'Unsupported Media Type'))
+            return func(*args, **kwargs)
+
+        return wrapped
+
+    return inner_decorator
```

Comment:
Be more precise in the name of the method, it only validate the content type header not the content, so `validate_content_type` is better